### PR TITLE
change sample mapping method from raw value to other units

### PR DESCRIPTION
**Pull request by kdw503**
- Previous
     For the piezo, raw value interval is fixed as `0..typemax(Int16)` and this is mapped to user specified `PIEZO_RANGES` value
     according to the `rig_key` as below
     `PIEZO_RANGES[rig_key] = (0.0μm .. 800.0μm, 0.0V .. 10.0V)`
     But, in this case, there is no way to map `typemin(Int16)..typemax(int16)` to `(-400.0μm .. 400.0μm, -10.0V .. 10.0V)` which
     is required for 'realm' microscope.
- Now
     We can specify the raw value also as
     `PIEZO_RANGES[rig_key] = (-32768..32767, -400.0μm .. 400.0μm, -10.0V .. 10.0V)`
     * note : raw range `typemin(Int16)..typemax(int16)` doesn't work. During the waveform validity check, it need to call 
                 `width(typemin(Int16)..typemax(int16))`. But, it return `0` different from expected value `65536`. But, the direct 
                 value `-32768..32767` works because the eltype of this range is not the `Int16` but `Int64` in this case.

### CODE REVIEW

File: rigs/dummy6002.jl [hunk 1]
```diff
@@ -53,7 +53,7 @@ LASER_OFF_TIME[rig_key] = 100 * Unitful.μs
 CAMERA_ON_TIME[rig_key] = 19526.0 * Unitful.ns #This is the worst jitter measured with Edge 5.5 and 4.2 cameras, see ImagineInterface issue #18
 CAMERA_OFF_TIME[rig_key] = 19526.0 * Unitful.ns #This is the worst jitter measured with Edge 5.5 and 4.2 cameras, see ImagineInterface issue #18
 
-PIEZO_RANGES[rig_key] = (0.0μm .. 400.0μm, 0.0V .. 10.0V)
+PIEZO_RANGES[rig_key] = (0..32767, 0.0μm .. 400.0μm, 0.0V .. 10.0V)
```
Comment (timholy) [inline rigs/dummy6002.jl:56]: For those that start at 0, should the top value be `Int(typemax(UInt16)) == 65535`? Or is the top bit still excluded?

**Review by timholy — COMMENTED** [review]
LGTM. Thanks!

File: rigs/dummy6002.jl (hunk 1 above)
Comment (kdw503) [reply rigs/dummy6002.jl:56]: Excluded I think, Actually, it looks NIDAQ maps Int16 values to -10 to 10V (it means -32768..32767 to -10..10V).
The previous code doesn't use full range. it only use positive values. It maps 0..32767 to 0..10V.

Comment (kdw503) [reply rigs/dummy6002.jl:56]: But pull request provide option to set the raw range freely other than fixed one 0..32767 for piezo.

Comment (timholy) [reply rigs/dummy6002.jl:56]: Thanks! Merge at will.

**Pull request merged**